bip-0137.mediawiki

```diff
@@ -0,0 +1,140 @@
+<pre>
+  BIP: 137
+  Layer: Applications
+  Title: Signatures of Messages using Private Keys
+  Author: Christopher Gilliard <christopher.gilliard@gmail.com>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0137
+  Status: Final
```

Comment:
> API/RPC and application layer BIPs must be implemented by at least two independent and compatible software applications.

Are there two?

Reply:
Yes.

Reply:
Which?

Reply:
Trezor, Ledger (partial support in their chrome app), Bitcoin Signature Tool, and any of the airdrops that supported these signatures like Bitcoin Rhodium, MWC, byte ball (I think). There are probably others that I don't know about.